src/ray/gcs/gcs_client_interface.h

```diff
@@ -80,9 +80,9 @@ class GcsClientInterface : public std::enable_shared_from_this<GcsClientInterfac
   /// Disconnect with GCS Service. Non-thread safe.
   virtual void Disconnect() = 0;
 
-  /// Get ActorStateAccessor for reading or writing or subscribing to
+  /// Get ActorStateAccessorInterface for reading or writing or subscribing to
   /// actors. This function is thread safe.
-  ActorStateAccessor &Actors() {
+  ActorStateAccessorInterface &Actors() {
```

Comment:
```suggestion
  ActorStateAccessor &Actors() {
```

We don't need to name this `Interface`, we can just call the implementations `<Type>ActorStateAccessor` like `RedisActorStateAccessor`

Reply:
ok